perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -1899,6 +1899,40 @@ public Application getApplicationById(PerunSession sess, int appId) throws Regis
 
 	}
 
+	@Override
+	public List<Application> getApplicationsForVo(PerunSession userSession, Vo vo, List<String> state, String dateFrom, String dateTo) throws PerunException {
+		vosManager.checkVoExists(userSession, vo);
+
+		//Authorization
+		if (!AuthzResolver.authorizedInternal(userSession, "getApplicationsForVo_Vo_List<String>_String_String_policy", Collections.singletonList(vo))) {
+			throw new PrivilegeException(userSession, "getApplicationsForVo");
+		}
+		try {
+			MapSqlParameterSource sqlParameterSource = new MapSqlParameterSource();
+			StringBuilder query = new StringBuilder(APP_SELECT);
+			query.append(" where a.vo_id=:voId");
+			sqlParameterSource.addValue("voId", vo.getId());
+			if (state != null && !state.isEmpty()) {
+				// list all
+				sqlParameterSource.addValue("states", state);
+				query.append(" and state in ( :states )");
+			}
+			if(dateFrom != null && !dateFrom.isEmpty()) {
+				sqlParameterSource.addValue("from", dateFrom);
+				query.append(" and and a.created_at::date >= date :from ");
+			}
+			if(dateTo != null && !dateTo.isEmpty()) {
+				sqlParameterSource.addValue("to", dateTo);
+				query.append(" and a.create_at::date <= date :to ");
```

Comment:
I think the correct column should be `created_at`

Reply:
Fixed.